docs/mesos/Vagrant.md

```diff
@@ -6,67 +6,31 @@
 > You are viewing the calico-containers documentation for release **release**.
 <!--- end of master only -->
 
-# Vagrant Deployed Mesos Cluster with Calico
-This guide will start a running Mesos cluster (master and two agents) with Calico Networking using a simple `vagrant up`.
+# Vagrant Start a Mesos Cluster with Calico
+This guide will show you how to use Vagrant to launch a Mesos Cluster
+with Calico installed and ready to network Docker Containerizer tasks.
 
-These machines are flexible and will allow you to use either the
-Docker Containerizer or the Unified Containerizer for launching tasks.
-
-## Prerequisites
+## 1. Install Dependencies
 This guide requires a host machine with:
 
  * [VirtualBox][virtualbox] to host the virtual machines.
  * [Vagrant][vagrant] to install and configure the machines in Virtual Box.
  * [Git][git]
 
-## Getting Started
-1. First, clone this repository and change to the Mesos vagrant directory:
-
-  ```
-  git clone https://github.com/projectcalico/calico-containers.git
-  cd calico-containers/docs/mesos/vagrant-centos
-  ```
+## 2. Download Demo Files
+   ```
+   git clone https://github.com/projectcalico/calico-containers.git
+   ```
 
-2. Then launch the Vagrant demo:
-  ```
-  vagrant up
-  ```
+## 3. Startup and SSH
```

Comment:
Ah - i see it's in the sub section below.  Perhaps the SSH instructions should be a separate sub section rather than in cluster layout